decorate.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './nameable'
```

Comment:
- [ ] good job changing this to require_relative, however, you are calling `nameable` but the file is called `namable` Therefore this will not work